src/commands/alias/set.ts

```diff
@@ -565,5 +565,12 @@ function handleCreateAliasError<T>(
     return 1;
   }
 
+  if (error instanceof ERRORS.DomainPermissionDenied) {
+    output.error(
+      `You don't have access to the domain ${error.meta.domain} under ${error.meta.context}.`
```

Comment:
Should we also reuse the `error.message` here?

Reply:
IMO no!

Reply:
but why? Isn't reusing `error.message` in accordance to the above mentioned new strategy of reusing and augmenting error messages where possible?